client/src/services/bms/types.ts

```diff
@@ -46,6 +46,9 @@ interface PersonalInformationResultData {
   lastName: string;
   email: string;
   clientId: string;
+  phone: string;
```

Comment:
How do the BMS handle optional fields? Are those empty strings, or null, or not present at all in the JSON? I'd rather they'd be declared as optional on our side, so they can be set as undefined if they don't have a value.

Reply:
yes